travis/run.sh

```diff
@@ -57,5 +57,11 @@ test -d $INST_TEST && rm -rf $INST_TEST
 make DESTDIR=$INST_TEST install
 find $INST_TEST
 
+case $(uname) in
+  Darwin)
+    pushd java && buck fetch :watchman :watchman-tests && buck test :watchman && popd
```

Comment:
This is currently failing on Travis, but the failure isn't bubbling up:

```
~/build/facebook/watchman/java ~/build/facebook/watchman
+buck fetch :watchman :watchman-tests
:::
::: '.nobuckcheck' file is present. Not updating buck.
:::
Using watchman.
[2016-07-13 11:48:02.716][error][command:64a216b7-9904-446f-9829-04a8155202fd][tid:14][com.facebook.buck.io.Watchman] Error 1 executing get-sockname
Warning raised by BUCK file parser: Traceback (most recent call last):
Warning raised by BUCK file parser:   File "/Users/travis/build/facebook/watchman/java/.buckd/tmp/buck_run.G6WC2K/buck4881761727118666697.py", line 12, in <module>
Warning raised by BUCK file parser:     import __builtin__
Warning raised by BUCK file parser: ImportError: No module named '__builtin__'
Parse error for build file /Users/travis/build/facebook/watchman/java/BUCK: Parser exited unexpectedly
```

I think you need to restructure it to be something like:

```
pushd java
buck fetch :watchman :watchman-tests || exit 1
buck test :watchman || exit 1
popd
```

and then figure out why it failed and resolve that :-)